feat: login, signup UI

harshitadaswani opened the pull request:
- added login and signup UI
- setup the respective routing of the pages

harshitadaswani committed:
feat: login, signup UI

ShreyashMahajan reviewed (approved):
Code is well structured, just one suggestion that try to create separate files for routes, i also got to know about this recently.
 ref - https://www.xenonstack.com/insights/reactjs-project-structure

dev-enforced commented on src/pages/Signup.jsx:
```diff
@@ -0,0 +1,42 @@
+
+                    <div className="flex-row flex-space-between p-xs m-xs">
+                        <div className="checkbox-div">
+                            <input type="checkbox" name="Remember-me" className="br-xs p-xs m-xs txt-m" />
```
Do pass an id-attribute in the input section having the value matching to the htmlFor attribute on the label later on so that whenever we click on the label that input section gets marked accordingly.

harshitadaswani replied:
done implemented

dev-enforced commented on src/App.js:
```diff
@@ -11,6 +11,8 @@ function App() {
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/products" element={<Products />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<Signup />} />
       </Routes>
```
CODE Suggestion:
Instead of adding routes in the App.js file, you could move it to a separate file say AppRoutes.js and then add the component here. This way, in the future, if the number of routes increases your code will look cleaner, readable easy to understand.

harshitadaswani replied:
done implemented

dev-enforced commented on src/pages/Login.jsx:
```diff
@@ -0,0 +1,44 @@
+
+                    <div className="flex-row flex-space-between">
+                        <div className="checkbox-div">
+                            <input type="checkbox" name="Remember-me" className="br-xs p-xs m-xs txt-m" />
```
Same as the suggestion mentioned in Signup section

harshitadaswani replied:
done

dev-enforced reviewed (approved):
The UI of Login and SignUp section looks good: https://deploy-preview-7--crypto-store.netlify.app/login
Liked the use of Utility classes which helped out a lot no use of external CSS files related to the component

harshitadaswani committed:
fix: suggestions by reviewers

- moved routes to a new component
- added id attributes to label and input elements
